### DISCUSSION TITLE
Set seeds disables spawn point function

### DISCUSSION THREAD

**Discussion by Rubiksmath** [Zodsmar/SeedSearcherStandaloneTool#120] [Q&A] [answered]
I really need to search through a list of set seeds (its a list of sister seeds of a specific seed), and to narrow down my search i need to have a specific spawn point, however it doesnt let me search for spawn point when I do this, and searching manually through all 65536 seeds in my txt would take me hundreds of hours, and I dont have that sort of time. Can you pretty please add the functionality of searching for spawn point from set seeds? i really dont understand why it specifically disables that feature when you click on "set seeds" anyway. Thanks.

**Comment by Zodsmar** [comment, marked as answer]
So I am currently working on updating to 1.18. I am doing a full rewrite of the code. The new version will have the ability to search around spawn point (So instead of 0,0 it can search around spawn) but are you talking about getting the coords for a spawn point in a seed?
